sdk/go/outbound_redis/internals.go

```diff
@@ -0,0 +1,66 @@
+package outbound_redis
+
+// #include "outbound-redis.h"
+import "C"
+import (
+	"errors"
+	"unsafe"
+)
+
+func publish(addr, channel string, payload []byte) error {
+	a := redisStr(addr)
+	c := redisStr(channel)
+	p := C.outbound_redis_payload_t{ptr: &payload[0], len: C.size_t(len(payload))}
```

Comment:
nit: could the variables here be a bit more explicit, please?
I can see it being very difficult to debug anyway, particularly given that it's almost entirely CGO.